### ISSUE TITLE
Social and Icon Widgets to allow for HTML "title" attribute and accessibility

### ISSUE THREAD

**Issue by acal** [pojome/elementor#5386]
- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

Social Icon Widgets should provide a default "title" attribute === to the social sharing service name and also a user defined title attribute. This would allow mouse overs to produce a nice title so that less obvious icons can be understood by users before clicking.

Furthermore, a screen reader class and text element should be inserted for accessibility reasons for social share and icon widgets. This is already done for Elementor Social Widget but not Share or Icon.

**Comment by black-eye** [comment]
+1

**Comment by ramiy** [comment]
For better accessibility, the "title" attributes are not required.

It can be implemented by adding a hidden text visible only for screen-readers. This method is implemented all over WordPress and across different Elementor widgets.

Current code:
```
<div class="elementor-share-btn elementor-share-btn_linkedin">
	<span class="elementor-share-btn__icon">
		<i class="fa fa-linkedin"></i>
	</span>
</div>
```

New Code:
```
<div class="elementor-share-btn elementor-share-btn_linkedin">
	<span class="elementor-share-btn__icon">
		<i class="fa fa-linkedin" aria-hidden="true" ></i>
		<span class="elementor-screen-only">Share on Linkedin</span>
	</span>
</div>
```

**Comment by ramiy** [comment]
Add new Merge Requests to Elementor Pro repo:

https://gitlab.com/elementor/elementor-pro/merge_requests/740

**Comment by ramiy** [comment]
Update: The Merge Requests has been accepted and merged to **Elenetor Pro 2.4**.

**Comment by Josh-Vaultoro** [comment]
This is making Yoast give a bad readability diagnosis because if you have a let's say 5 share buttons you have
"Consecutive sentences: The text contains 5 consecutive sentences starting with the same word. Try to mix things up!"
this is because you have 
Share on facebook
Share on Twitter
Share on Linkedin...

It would be nice to be able to change these screen readable texts in the elementary GUI so remove these issues.
